TBG/DebugUtils.h

```diff
@@ -1,9 +1,35 @@
 #pragma once
 
+// the following are UBUNTU/LINUX, and MacOS ONLY terminal color codes.
 #include <string>
+#define RESET "\033[0m"
+#define BLACK "\033[30m"			  /* Black */
+#define RED "\033[31m"				  /* Red */
+#define GREEN "\033[32m"			  /* Green */
+#define YELLOW "\033[33m"			  /* Yellow */
+#define BLUE "\033[34m"				  /* Blue */
+#define MAGENTA "\033[35m"			  /* Magenta */
+#define CYAN "\033[36m"				  /* Cyan */
+#define WHITE "\033[37m"			  /* White */
+#define BOLDBLACK "\033[1m\033[30m"	  /* Bold Black */
+#define BOLDRED "\033[1m\033[31m"	  /* Bold Red */
+#define BOLDGREEN "\033[1m\033[32m"	  /* Bold Green */
+#define BOLDYELLOW "\033[1m\033[33m"  /* Bold Yellow */
+#define BOLDBLUE "\033[1m\033[34m"	  /* Bold Blue */
+#define BOLDMAGENTA "\033[1m\033[35m" /* Bold Magenta */
+#define BOLDCYAN "\033[1m\033[36m"	  /* Bold Cyan */
+#define BOLDWHITE "\033[1m\033[37m"	  /* Bold White */
+// Note: If you don't use RESET the color will remain changed until the next time you use a color code.
+// source: https://stackoverflow.com/questions/9158150/colored-output-in-c
 
 namespace DebugUtils
 {
-	void Log(std::string message);
+	enum Severity
```

Comment:
you can put this enum outside of the namespace. it'll only be included with the header file, so there will be no collisions. and if we ever need a severity enum, we'll just use/add this one